lib/incito-browser/incito.js

```diff
@@ -368,8 +380,8 @@ export default class Incito extends MicroEvent {
             } else {
                 attrs.src = view.src;
             }
-        } else if (view.view_name === 'IncitoEmbedVideo') {
-            classNames.push('incito__incito-embed-video');
+        } else if (view.view_name === 'IncitoEmbedView') {
```

Comment:
Is this removal of `IncitoEmbedVideo` views cleared with Incito team?

Reply:
@klarstrup That was a typo by me (I wrote `IncitoEmbedVideo` when I meant `IncitoEmbedView`). We still support `VideoEmbedView` and `HTMLEmbedView`.